README.md

```diff
@@ -1,5 +1,14 @@
 # Observe API
 
+Data service for [Observe](https://github.com/developmentseed/observe) platform. This is a web server built in Node.js that provides an application programming interface for the [dashboard](https://github.com/developmentseed/observe-dashboard) and [mobile app](https://github.com/developmentseed/observe) of Observe.
+
+Documentation is available at:
+
+- [API](http://developmentseed.github.io/observe-api)
+- [Platform](https://github.com/developmentseed/observe#documentation) 
+
+This software is in early stages of development. Please visit the [issues page](https://github.com/developmentseed/observe-api/issues) to learn about known bugs and features development.
```

Comment:
Very minor grammatical nitpick: `learn about known bugs and features development` -> `learn about known bugs and feature development`